MiniAn/minian_parameters.py

```diff
@@ -0,0 +1,319 @@
+
+        noise_freq: float   = self.parameters[defs.Parameters.danse.NOISE_FREQ]
+        thres_corr: float   = self.parameters[defs.Parameters.danse.THRES_CORR]
+
+        self.params_LocalCluster = {
+            "n_workers": 4,
+            "memory_limit": "auto",
+            "resources": {"MEM": 1}, # constrain the number of tasks that can be concurrently in memory for each worker
+            "threads_per_worker": 2,
+            "dashboard_address": ":8787",
+            "local_directory": self.paths[defs.Parameters.Path.TMP_DIR]
+        }
+
+        self.params_load_doric = {
+            "fname": self.paths[defs.Parameters.Path.FILEPATH],
+            "h5path": self.paths[defs.Parameters.Path.H5PATH],
+            "dtype": np.uint8,
+            "downsample": {"frame": self.parameters[defs.Parameters.danse.TEMPORAL_DOWNSAMPLE] if not self.preview else self.preview_parameters[defs.Parameters.Preview.TEMPORAL_DOWNSAMPLE],
+                            "height": self.parameters[defs.Parameters.danse.SPATIAL_DOWNSAMPLE],
+                            "width": self.parameters[defs.Parameters.danse.SPATIAL_DOWNSAMPLE]},
+            "downsample_strategy": "subset",
+        }
+
+        self.params_save_minian = {
+            "dpath": os.path.join(self.paths[defs.Parameters.Path.TMP_DIR], "final"),
+            "meta_dict": {"session": -1, "animal": -2},
+            "overwrite": True,
+        }
+
+        self.params_get_optimal_chk = {
+            "dtype": float
+        }
+
+        self.params_denoise = {
+            'method': 'median',
+            'ksize': mn_utils.round_down_to_odd(neuron_diameter[-1]/2.0) # half of the maximum diameter
+        }
+
+        self.params_remove_background = {
+            'method': 'tophat',
+            'wnd': np.ceil(neuron_diameter[-1]) # largest neuron diameter
+        }
+
+        self.params_estimate_motion = {
+            'dim': 'frame'
+        }
+
+        self.params_apply_transform = {
+            'fill': 0
+        }
+
+        wnd = 60 # time window of 60 seconds
+        self.params_seeds_init = {
+            'wnd_size': self.fr*wnd,
+            'method': 'rolling',
+            'stp_size': self.fr*wnd / 2,
+            'max_wnd': neuron_diameter[-1],
+            'diff_thres': 3
+        }
+
+        self.params_pnr_refine = {
+            "noise_freq": noise_freq,
+            "thres": 1
+        }
+
+        self.params_ks_refine = {
+            "sig": 0.05
+        }
+
+        self.params_seeds_merge = {
+            'thres_dist': neuron_diameter[0],
+            'thres_corr': thres_corr,
+            'noise_freq': noise_freq
+        }
+
+        self.params_initA = {
+            'thres_corr': thres_corr,
+            'wnd': neuron_diameter[-1],
+            'noise_freq': noise_freq
+        }
+
+        self.params_unit_merge = {
+            'thres_corr': thres_corr
+        }
+
+        self.params_get_noise_fft = {
+            'noise_range': (noise_freq, 0.5)
```

Comment:
I just noticed -- what happens if `noise_freq > 0.5`? @kapilseh @4D42 
Does it make sense to set it more than 0.5? Do you know why it's 0.5? Is it possible to set noise_freq more than 0.5 in our view?

Reply:
I don't know why it is 0.5 . I assume it come from the default pipeline. _In pipeline (param_get_noise = {"noise_range": (0.06, 0.5)})_

In the view it is possible to set `noise_freq > 0.5` . So If I set it `noise_freq > 0.5`  sometime (depending of the values) it don't fine any cells.

Maybe we should make it not possible to set `noise_freq > 0.5`. Or change it to be `'noise_range': (noise_freq, 8 * noise_freq)` or `'noise_range': (noise_freq, noise_freq + 0.5)` something like that. Or maybe just reset it to the default `"noise_range": (0.06, 0.5)`

Reply:
@kapilseh would you know why it's 0.5?

Reply:
so if 0.5 represents nyquist frq. We need to make the following changes-
1. the max possible value for noise frq (ui->dsb_minian_noise)  should be - sampling rate /4 
2. Also 'noise_range': (noise_freq, sampling rate/4)